packages/frontend/src/lib/RecipeDetails.svelte

```diff
@@ -53,27 +49,55 @@ const toggle = () => {
       </div>
 
       <div class="w-full bg-charcoal-600 rounded-md p-4">
-        {#if recipeStatus !== undefined && recipeStatus.tasks.length > 0}
-          {#if recipeStatus.state === 'error'}
-            <Button on:click="{() => onPullingRequest()}" class="w-full p-2" icon="{faRefresh}">Retry</Button>
-          {:else if recipeStatus.state === 'loading' || recipeStatus.state === 'running'}
-            <Button inProgress="{true}" class="w-full p-2" icon="{faPlay}">
-              {#if recipeStatus.state === 'loading'}Loading{:else}Running{/if}
-            </Button>
+        <div class="flex flex-row items-center">
+          {#if envState}
+            <div class="grow whitespace-nowrap overflow-hidden text-ellipsis text-sm text-gray-300">{envState.pod.Name}</div>
+          {:else}
+            <div class="grow whitespace-nowrap overflow-hidden text-ellipsis text-sm text-gray-500 italic">(no pod running)</div>
           {/if}
-        {:else}
-          <Button on:click="{() => onPullingRequest()}" class="w-full p-2" icon="{faPlay}">Run application</Button>
-        {/if}
-
-        <div class="text-xs text-gray-700 mt-3">
-          This will git clone the application, download the model, build images, and run the application as a pod
-          locally.
+          <div class="shrink-0">
+            <EnvironmentControls recipeId={recipeId} object={envState} tasks={recipeStatus?.tasks} />
+          </div>  
         </div>
         {#if recipeStatus !== undefined && recipeStatus.tasks.length > 0}
           <div class="mt-4 text-sm font-normal py-2">
             <TasksProgress tasks="{recipeStatus.tasks}" />
           </div>
         {/if}
+        <!--div class="flex flex-row">
```

Comment:
remove comments ?

Reply:
yes, I'll do some cleanup when I'm sure I don't need anything from the commented code